src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Models/DelegatedCodeActionParams.cs

```diff
@@ -7,8 +7,11 @@
 namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Models;
 
 [DataContract]
-internal class DelegatedCodeActionParams : CodeActionParams
+internal class DelegatedCodeActionParams
 {
-    [DataMember(Name = "_vs_hostDocumentVersion")]
+    [DataMember(Name = "_razor_hostDocumentVersion")]
```

Comment:
Seemed questionable to use the `_vs` prefix when we're not VS.

Reply:
Realised now that this doesn't inherit from anything, these don't need any crazy prefix, as they can't possibly clash. Will update